src/app/pages/about/about.component.html

```diff
@@ -57,6 +57,13 @@ <h1 class="vf-text vf-text-heading--4">Eligibility Criteria</h1>
               the criteria for published data (see below), or a supporting
               justification as to why it should be included in the HCA.
             </li>
+            <li>
+              Be available in raw fastq format and/or as a gene expression matrix
+              (raw counts or processed counts). Multiplexed data must be demultiplexed
```

Comment:
Language based on criteria for inclusion rather than the action "upload" might work better. So instead of "upload", I would be tempted to say "prior to inclusion". I've taken a stab at a suggestion here and below...

```suggestion
              (raw counts or processed counts). Data from multiple samples must be
```